### DISCUSSION TITLE
Is there any future plan to add getter in unMount?

### DISCUSSION THREAD

**Discussion by dante01yoon** [pmndrs/jotai#1571] [Ideas] [answered]
Hi, I read this issues  #691,  I wonder if there can be support using getter in unMount in future release.

derived atom could be handy, but in some cases, I guess declare multiple atoms for just reference some states could be cumbersome in some cases.

Any thoughts?

**Comment by dai-shi** [comment, marked as answer]
I have tried it when I first implemented `.onMount`. It was very weird implementation, and didn't feel like maintainable. It's our design decision to make the library be as primitive as possible.
So, we prefer multiple atoms.
If developer experience is not nice with what's provided, it's a field for `jotai/utils` or 3rd party packages.
